### PR TITLE
add realization of reverse string

### PR DESCRIPTION
**Pull request by AnnLvu**

### CODE REVIEW

File: src/main/java/main/StringReverse.java [hunk 1]
```diff
@@ -1,15 +1,47 @@
 package main;
 
+import java.sql.SQLOutput;
```
Comment (KITKATJN) [inline src/main/java/main/StringReverse.java]: Do you really need this import?
- [ ] remove all unused imports

File: src/main/java/main/StringReverse.java [hunk 2]
```diff
@@ -1,15 +1,47 @@
...
+        reverseArray(charArray);
+        printArray(charArray);
+        return new String(charArray);
+    }
+
+
+    public static void reverseArray(char[] charArray) {
+        {
+            for (int i = charArray.length - 1; i >= 0; i--) {
+                for (int j = 0; j < i; j++) {
```
Comment (KITKATJN) [inline src/main/java/main/StringReverse.java]: - [ ] Please rewrite this method, using only one cycle "for"

File: src/main/java/main/StringReverse.java [hunk 3]
```diff
@@ -1,15 +1,47 @@
...
     public static String stringReverse(String stringToReverse) {
-        // insert your code here
-        // you need to convert string to array of chars and work with it
-        return stringToReverse;
+        if (stringToReverse == null) {
+            return null;
+        }
+
+        char[] charArray = stringToReverse.toCharArray();
+        reverseArray(charArray);
```
Comment (KITKATJN) [inline src/main/java/main/StringReverse.java]: you can make "reverseArray" returning String

File: src/main/java/main/StringReverse.java [hunk 4]
```diff
@@ -1,15 +1,47 @@
...
+        printArray(charArray);
+        return new String(charArray);
+    }
+
+
+    public static void reverseArray(char[] charArray) {
```
Comment (KITKATJN) [inline src/main/java/main/StringReverse.java]: - [ ] change void to String

File: src/main/java/main/StringReverse.java (hunk 1 above)
Comment (AnnLvu) [reply src/main/java/main/StringReverse.java]: removed

File: src/main/java/main/StringReverse.java (hunk 2 above)
Comment (AnnLvu) [reply src/main/java/main/StringReverse.java]: rewrite

File: src/main/java/main/StringReverse.java (hunk 3 above)
Comment (AnnLvu) [reply src/main/java/main/StringReverse.java]: made "reverseArray" returning String

File: src/main/java/main/StringReverse.java (hunk 4 above)
Comment (AnnLvu) [reply src/main/java/main/StringReverse.java]: change

**Pull request closed without merging**